lib/phlex/view.rb

```diff
@@ -178,13 +178,20 @@ def helpers
 			@_view_context
 		end
 
+		def resolve_attributes(attributes)
+			attributes
+		end
+
 		def _attributes(attributes, buffer: +"")
 			if attributes[:href]&.start_with?(/\s*javascript/)
 				attributes[:href] = attributes[:href].sub(/^\s*(javascript:)+/, "")
 			end
 
+			attributes = resolve_attributes(attributes)
```

Comment:
I think `tailor_attributes` might be a better name for this, “to make or adapt for a particular purpose”.

Unfortunately, we have performance to contend with once again. A method call or `responds_to?` check will add about 1% overhead, which is quite a lot if you never use the feature.

Perhaps this could go in a module, like the callbacks.

Reply:
There is no `responds_to?` call here.

Reply:
I know, but that's the alternative to a method call. They're both about 1% I believe.

Reply:
Oh right, I get ya. 

Yeah, a module could be better actually, as it makes it completely opt in, with no perf hit if not used.

Reply:
The only problem is we can't make a module that overrides this without having to duplicate the rest of the method in that module.

Reply:
Then I see the only option being to always call `resolve_attributes`

Reply:
There is another option that doesn't cost anything if you don't use it. We could rename `_attributes` to `resolve_attributes` or something like that and let you override it. You’d have to pass your result to `super` but I think that's reasonable for something like this.

Reply:
Ok, so `resolve_attributes`, `attributes`, `tailor_attributes`? Maybe just `attributes`?